OJP/OJP_FareSupport.xsd

```diff
@@ -137,45 +137,331 @@
 	<xs:annotation>
 		<xs:documentation>========================================== Booking ==========================================</xs:documentation>
 	</xs:annotation>
-	<xs:complexType name="BookingArrangementStructure">
+	<xs:simpleType name="ContactTypeEnumeration">
 		<xs:annotation>
-			<xs:documentation>[a more generalised form of BOOKING ARRANGEMENTS in TMv6] arrangement for booking any leg or legs of a journey </xs:documentation>
+			<xs:documentation>Allowed values for CONTACT role. +v1.2.2</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:NMTOKEN">
+			<xs:enumeration value="any"/>
+			<xs:enumeration value="information"/>
+			<xs:enumeration value="reservations"/>
+			<xs:enumeration value="lostProperty"/>
+			<xs:enumeration value="publicRelations"/>
+			<xs:enumeration value="complaints"/>
+			<xs:enumeration value="emergency"/>
+			<xs:enumeration value="other"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="PhoneType">
+		<xs:annotation>
+			<xs:documentation>Type for an International phone number. + Country + MSIDDN e.g. +41 675601 etc.</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:normalizedString"/>
+	</xs:simpleType>
+	<xs:simpleType name="EmailAddressType">
+		<xs:annotation>
+			<xs:documentation>Email address type a@b.</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:normalizedString"/>
+	</xs:simpleType>
+	<xs:complexType name="ContactDetailsStructure" abstract="false">
+		<xs:annotation>
+			<xs:documentation>Type for contact details.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element name="BookingAgencyName" type="InternationalTextStructure" minOccurs="0">
+			<xs:element name="ContactPerson" type="InternationalTextStructure" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Name of the booking agency (contractual partner).</xs:documentation>
+					<xs:documentation>Name of contact person.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="BookingUrl" type="WebLinkStructure" minOccurs="0">
+			<xs:element name="Email" type="EmailAddressType" minOccurs="0">
```

Comment:
ContactPerson is an InternationalTextStructure, so I would think that we could have different contacts for different languages. Shouldn't we as well be able to have different phone numbers and e-mail adresses for different languages? E.g., by basing them on InternationalTextStructure as well?